NEWS

```diff
@@ -1,6 +1,11 @@
 shiny-server 1.5.0
 --------------------------------------------------------------------------------
-* Add auto-reconnect capabilities.
+* Add auto-reconnect capabilities. Can be disabled via `disable_reconnect`
```

Comment:
Oh. Note that it's optional. Meaning that the preferred/recommended way to reference it as of the next release when it's on by default will be

```
disable_reconnect;
```

with no parameter, which feels pretty clean. It's just cludgy now since it's temporarily opt-in.